bob/src/cluster/simple.rs

```diff
@@ -38,34 +38,33 @@ impl Quorum {
         }
         keys_by_nodes
     }
-}
 
-#[async_trait]
-impl Cluster for Quorum {
```

Comment:
I just noticed, that types in `quorum.rs` and `simple.rs` have same name. How does it compile without errors?

Reply:
Different modules, Quroum is renamed as SimpleQuorum

Reply:
I don't see, that rename done

Reply:
Renamed here https://github.com/qoollo/bob/blob/master/bob/src/cluster/mod.rs#L10